src/com/dynatrace/loadrunner/logic/CFilePatcher.java

```diff
@@ -78,18 +78,18 @@ private void handleInstruction(String instruction, String commentedInstruction)
 		if (write==null)
 			write=instruction;
 		write=removeEOF(commentedInstruction);
-		if (!instruction.contains(Constants.ADD_HEADER_FUNCTION))
+		if (!instruction.contains(Constants.ADD_HEADER_FUNCTION_TEST))
```

Comment:
if we want to publish that and don't want to shame of the implementation - it definitely needs refactor (even if it's not part of your changes)
I mean - the whole class and code organization; we can do it next week in pair-programming